nanomq/pub_handler.c

```diff
@@ -1555,10 +1560,12 @@ decode_pub_message(nano_work *work, uint8_t proto)
 		used_pos = pos;
 
 		if (MQTT_PROTOCOL_VERSION_v5 == proto) {
+			if (work->proto == PROTO_MQTT_BRIDGE)
+				is_copy = true;
 			pub_packet->var_header.publish.properties =
 			    decode_properties(msg, &pos,
 			        &pub_packet->var_header.publish.prop_len,
-			        false);
+			        is_copy);
```

Comment:
sure there is no side effect like memory leak?

Reply:
> sure there is no side effect like memory leak?

no leaks. Mr.Alvin also did the free part.